### DISCUSSION TITLE
Make tab sizes customizable within mobile app

### DISCUSSION THREAD

**Discussion by cwhakes** [community/community#3836] [Mobile]
Right now, there does not appear to be a setting to modify tab width within the moblie app. On the platform where space is at it's most premium, tab size is locked in at 8 character-widths.

The mobile app doesn't even respect the setting within `.editorconfig`. However, while that setting should be followed by default, it still should be overidable. 4 character-widths may be fine on a 32" monitor and still be too large on a 5.5" screen.

**Comment by nplasterer** [comment]
Thanks for your feedback. We're looking into ways to make this better on mobile.

**Reply by barnwalp** [reply]
Is there any solution available now for the code indentation problem on github android app?

**Reply by Fb9027** [reply]
Ok

**Reply by Fb9027** [reply]
Ok

**Reply by Fb9027** [reply]
Which app to iOS?

**Reply by Phoenix616** [reply]
> Thanks for your feedback. We're looking into ways to make this better on mobile.

Any news on this? Having them 8 wide by default is just insane. At least use 4 as the default...

**Reply by gioragutt** [reply]
Up. This makes code samples in readmes unreadable.

**Comment by rfberaldo** [comment]
This is very important to have, it defaults to 8 (why?) spaces for a single tab, code gets huge horizontally.

An option within the 3 dots when browsing code should be perfect imo.

**Reply by binury** [reply]
Tab space of 8 default is crazy for mobile! 

![image](https://github.com/user-attachments/assets/0201e7f7-b363-4a38-8b6f-c6c2f97d8807)

![image](https://github.com/user-attachments/assets/a4fefda8-2f9a-492c-b778-4ccda012e45e)

**Comment by JPHutchins** [comment]
Microsoft should be embarrassed by this. Not only is the tab size in markdown rendered code blocks not configurable (along with tab size in general), but it defaults to 8, an unpopular default size on a PC, let alone on mobile.

**Reply by kakol20** [reply]
This has nothing to do with Microsoft
